mullvad-daemon/src/device/mod.rs

```diff
@@ -1437,24 +1435,26 @@ mod test {
         );
     }
 
-    /// Test whether the relay selector selects wireguard often enough, given no special
-    /// constraints, to verify that the device is valid
-    #[test]
-    fn test_validates_by_default() {
-        for attempt in 0.. {
-            let should_validate =
-                TunnelStateChangeHandler::should_check_validity_on_attempt(attempt);
-            let (_, _, tunnel_type) =
-                RelaySelector::preferred_tunnel_constraints(attempt.try_into().unwrap());
-            assert_eq!(
-                tunnel_type,
-                TunnelType::Wireguard,
-                "failed on attempt {attempt}"
-            );
-            if should_validate {
-                // Now that we've triggered a device check, we can give up
-                break;
-            }
-        }
-    }
+    // TODO(markus): `preferred_tunnel_constraints` is slated for removal - consider writing a new test which
```

Comment:
Don't forget to remove of update this commented out test

Reply:
Done!